Part1.md

```diff
@@ -1,130 +1,151 @@
+5.  You can see which flights you've booked on the User page. Click on the **User** button o go to the **User** page. 
+<br> <br> 
 ![](assets/travel-app-screen-user.png)
 
-To search for an hotel, navigate to the hotel page:
+### **To Find a Hotel**
 
-Click on the "earth" icon in the navigation bar (rightmost icon). Click on the Find Hotels button if you want to have a look at any first 100 hotels. Refine your search using the input boxes, either by keyword, geographically or both.
+ **Warning**: You should make sure that you've installed Couchbase 4.5 with FTS enabled. The API backend that's used by the hotels page uses an FTS index that you have to create, named hotels.
 
-![](assets/travel-app-screen-hotels.png)
+1. Click on the **earth icon** button in the top-right navigation bar to go to the **Find Hotels** page. 
+2. Click on the **Find Hotels** button to see the first 100 hotel results.<BR> **Note:** You can refine your search using the input fields, either by a **Description** keyword, by **Loction** or both.
+<br><br>
+![](assets/travel-app-screen-hotels_700.png)
 
 > Warning: You should make sure that you have installed Couchbase 4.5 with FTS enabled. The API backend that is used by the hotels page makes use of an FTS index that you have to create, named hotels.
 
-## Using CBQ
+## **Using CBQ**
 
-You can browse and access documents in Couchbase using browser and command line tools without writing any code. This can be helpful to you if you simply wish to inspect cluster data without writing code. You can access individual documents using the command line and web console.
+You can browse and access documents in Couchbase without writing any code, by using browser and command line tools. This is useful if you want to look at  cluster data without writing code. You can access individual documents using the command line and web console.
 
-### Using the command-line query shell (cbq)
+### **Using the command-line query shell (cbq)**
 
-You can use the cbq program as a command line query shell to issue N1QL queries in couchbase. cbq is available on all cluster nodes and does not require a separate installation. Simply run cbq from the Couchbase installation directory ( /opt/couchbase/bin/cbq on Linux and "/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/cbq"). Once in the shell, you should be greeted by a prompt and ready to issue any N1QL query.
+You can use the cbq program as a command line query shell to issue N1QL queries in couchbase.
+
+ cbq is available on all cluster nodes and doesn't require a separate installation. <br>Simply run cbq from the Couchbase installation directory ( `/opt/couchbase/bin/cbq` on Linux and `/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/cbq`). Once in the shell, you'll be greeted by a prompt and ready to issue any N1QL query, such as: 
 
 ```bash
-$ ./cbq
-cbq> select airportname FROM `travel-sample` WHERE airportname IS NOT NULL LIMIT 1;
-	// TODO
```

Comment:
It would be brilliant if you could run this command on your machine to replace that todo comment with what's missing.